docs/web3.eth.account.rst

```diff
@@ -0,0 +1,362 @@
+          'ciphertext': 'd6dbb56e4f54ba6db2e8dc14df17cb7352fdce03681dd3f90ce4b6c1d5af2c4f',
+          'kdf': 'pbkdf2',
+          'kdfparams': {'c': 1000000,
+           'dklen': 32,
+           'prf': 'hmac-sha256',
+           'salt': '45cf943b4de2c05c2c440ef96af914a2'},
+          'mac': 'f5e1af09df5ded25c96fcf075ada313fb6f79735a914adc8cb02e8ddee7813c3'},
+         'id': 'b812f3f9-78cc-462a-9e89-74418aa27cb0',
+         'version': 3}
+
+        >>> w3.eth.account.decrypt(encrypted, 'correcthorsebatterystaple')
+        b"\xb2\\}\xb3\x1f\xee\xd9\x12''\xbf\t9\xdcv\x9a\x96VK-\xe4\xc4rm\x03[6\xec\xf1\xe5\xb3d"
+
+
+.. py:method:: Account.sign(message=None, private_key=None, message_hexstr=None, message_text=None)
+
+    Sign the message provided. This is equivalent to :meth:`web3.eth.Eth.sign` but with
+    a local private key instead of an account in a connected client.
+    
+    Caller must supply exactly one of the message types:
+    in bytes, a hex string, or a unicode string. The message will automatically
+    be prepended with the text indicating that it is a message (preventing it
+    from being used to sign a transaction). The prefix is: ``b'\x19Ethereum Signed Message:\n'``
+
+    :param message: the message message to be signed
+    :type message: bytes or int
+    :param private_key: the key to sign the message with
+    :type private_key: hex str, bytes or int
+    :param str message_hexstr: the message encoded as hex
+    :param str message_text: the message as a series of unicode characters (a normal Py3 str)
+    :returns AttributeDict: Various details about the signature - most
+      importantly the fields: v, r, and s
+
+    .. code-block:: python
+
+        >>> msg = "I♥SF"
+        >>> key = "\xb2\\}\xb3\x1f\xee\xd9\x12''\xbf\t9\xdcv\x9a\x96VK-\xe4\xc4rm\x03[6\xec\xf1\xe5\xb3d"
+        >>> w3.eth.account.sign(message_text=msg, private_key=key)
+        {'message': b'I\xe2\x99\xa5SF',
+         'messageHash': '0x1476abb745d423bf09273f1afd887d951181d25adc66c4834a70491911b7f750',
+         'r': '0xe6ca9bba58c88611fad66a6ce8f996908195593807c4b38bd528d2cff09d4eb3',
+         's': '0x3e5bfbbf4d3e39b1a2fd816a7680c19ebebaf3a141b239934ad43cb33fcec8ce',
+         'signature': '0xe6ca9bba58c88611fad66a6ce8f996908195593807c4b38bd528d2cff09d4eb33e5bfbbf4d3e39b1a2fd816a7680c19ebebaf3a141b239934ad43cb33fcec8ce1c',
+         'v': 28}
+
+        # these are all equivalent:
+        >>> w3.eth.account.sign(w3.toBytes(text=msg), key)
+        >>> w3.eth.account.sign(bytes(msg, encoding='utf-8'), key)
+
+        >>> Web3.toHex(text=msg)
+        '0x49e299a55346'
+        >>> w3.eth.account.sign(message_hexstr='0x49e299a55346', private_key=key)
+        >>> w3.eth.account.sign(0x49e299a55346, key)
+
+
+.. py:method:: Account.recoverMessage(data=None, hexstr=None, text=None, vrs=None, signature=None)
+
+    Get the address of the account that signed the message with the given hash.
```

Comment:
copy/paste

Looks like this should be *"Get the address of the account that signed the given message"*.